src/main/java/com/project/daechiwon/domain/community/repository/CommunityUserRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.project.daechiwon.domain.community.repository;
+
+import com.project.daechiwon.domain.community.entity.CommunityUser;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CommunityUserRepository extends CrudRepository<CommunityUser, Long> {
```

Comment:
I heard that `CommunityUser` uses persistence cascade from `Community`..
So Why do you use repository even you use persistence cascade?

It's a ridiculous;;
Do not waste your time and memory.. :(

It's like Jindong Huh!